docs/modules/ROOT/pages/list-of-metrics.adoc

```diff
@@ -94,18 +94,24 @@ in milliseconds).
 
 |status
 |Information regarding the job's status. Potential values:
-0 - submitted;
-1 - initialization phase;
-2 - running;
-3 - suspended;
-4 - exporting snapshot;
-5 - completing phase;
-6 - failed;
-7 - completed successfully;
+
+* 0 - submitted
+* 1 - initialization phase
+* 2 - running
+* 3 - suspended
+* 4 - exporting snapshot
+* 5 - completing phase
+* 6 - failed
+* 7 - completed successfully
 
 |userCancelled
-|Details on whether the user canceled the job. Potential values: 1 - canceled by the user, 0 - otherwise.
+|Details on whether the user canceled the job. Potential values:
+
+* 1 - canceled by the user
+* 0 - otherwise
```

Comment:
```suggestion
* 0 - not canceled by user
```

Reply:
0 may mean: running, finished successfully, failed due to error

Reply:
Addressed in https://github.com/hazelcast/hz-docs/pull/1056/commits/6fb9b1f9abe4a03c8864f830398a4ac7b2b46151